internal/characters/cyno/cons.go

```diff
@@ -0,0 +1,156 @@
+package cyno
+
+import (
+	"github.com/genshinsim/gcsim/pkg/core/attributes"
+	"github.com/genshinsim/gcsim/pkg/core/combat"
+	"github.com/genshinsim/gcsim/pkg/core/event"
+	"github.com/genshinsim/gcsim/pkg/core/player/character"
+	"github.com/genshinsim/gcsim/pkg/modifier"
+)
+
+const (
+	c1Key = "cyno-c1"
+	c6Key = "cyno-c6"
+)
+
+// After using Sacred Rite: Wolf's Swiftness, Cyno's Normal Attack SPD will be increased by 20% for 10s.
+// If the Judication effect of his Passive Talent Featherfall Judgment is triggered during Secret Rite: Chasmic Soulfarer,
+// the duration of this increase will be refreshed.
+//
+// You need to unlock the Passive Talent "Featherfall Judgment."
+func (c *char) c1() {
+	m := make([]float64, attributes.EndStatType)
+	m[attributes.AtkSpd] = 0.2
+	c.AddAttackMod(character.AttackMod{
+		Base: modifier.NewBaseWithHitlag(c1Key, 600), // 10s
+		Amount: func(atk *combat.AttackEvent, t combat.Target) ([]float64, bool) {
+			if atk.Info.AttackTag != combat.AttackTagNormal {
+				return nil, false
+			}
+			return m, true
+		},
+	})
+}
+
+// When Cyno's Normal Attacks hit opponents, his Electro DMG Bonus will
+// increase by 10% for 4s. This effect can be triggered once every 0.1s. Max 5
+// stacks.
+func (c *char) c2() {
+	const c2Key = "cyno-c2"
+	const c2Icd = "cyno-c2-icd"
+	stacks := 0
+	m := make([]float64, attributes.EndStatType)
+	c.Core.Events.Subscribe(event.OnDamage, func(args ...interface{}) bool {
+		atk := args[1].(*combat.AttackEvent)
+		if atk.Info.ActorIndex != c.Index {
+			return false
+		}
+		if c.StatusIsActive(c2Icd) {
+			return false
+		}
+		if atk.Info.AttackTag != combat.AttackTagNormal {
+			return false
+		}
+
+		if !c.StatusIsActive(c2Key) {
```

Comment:
StatModIsActive would be more accurate (it's just an alias for the same function)